inc/nvapi.h

```diff
@@ -19739,6 +19856,7 @@ typedef enum
 {
     OUT_OF_BAND_RENDER  = 0,
     OUT_OF_BAND_PRESENT = 1,
+    OUT_OF_BAND_IGNORE  = 2,
 } NV_OUT_OF_BAND_CQ_TYPE;

```

Comment:
We are casting this to https://github.com/HansKristian-Work/vkd3d-proton/blob/master/include/vkd3d_vk_includes.h#L48

Reply:
Hi @esullivan-nvidia , do you know what needs to be done here when `OUT_OF_BAND_IGNORE` has been provided as argument in `NvAPI_D3D12_NotifyOutOfBandCommandQueue`?

Reply:
Handling this with an early bail out for now https://github.com/jp7677/dxvk-nvapi/pull/188/commits/5c277564589bf62c50b9ac4da2bc4c7d370d8c2a

Reply:
Sorry, I just saw this in my inbox. I think the early bail out for now is the best approach. We don't have any concept of this out of band queue type in LL2 so there isn't much we can do anyways. I will get in touch if we need to make any additional changes.

Reply:
Thanks for your confirmation!